src/systemdhook.c

```diff
@@ -517,6 +523,11 @@ static int prestart(const char *rootfs,
 			return -1;
 		}
 
+                /* Change uid and gid to something the container can handle */
+                if (chown(cont_journal_dir, uid, gid) == -1) {
+                        pr_perror("Failed to set container journal directory owner: %s, errno: [%d]", cont_journal_dir, errno);
```

Comment:
This is the error that I talked about in my e-mail.  When it tries to set the systemd_named_path, it fails everytime.